ert3/console/_console.py

```diff
@@ -113,6 +113,20 @@ def _build_clean_argparser(subparsers: Any) -> None:
     )
 
 
+def _build_start_storage_argparser(subparsers: Any) -> None:
+    start_storage_parser = subparsers.add_parser(
```

Comment:
Is `start_storage_parser` variable used somewhere? If not I wouldn't create it ...